src/component/Download/FinishedCard.js

```diff
@@ -329,61 +327,58 @@ export default function FinishedCard(props) {
-                                                >
-                                                    <Typography
-                                                        className={
-                                                            classes.subFileName
-                                                        }
-                                                    >
-                                                        <TypeIcon
-                                                            className={
-                                                                classes.subFileIcon
-                                                            }
-                                                            fileName={
-                                                                value.path
-                                                            }
-                                                        />
-                                                        {value.path}
-                                                    </Typography>
-                                                </TableCell>
-                                                <TableCell
-                                                    component="th"
-                                                    scope="row"
-                                                >
-                                                    <Typography noWrap>
-                                                        {" "}
-                                                        {sizeToString(
-                                                            value.length
-                                                        )}
-                                                    </Typography>
-                                                </TableCell>
-                                                <TableCell
-                                                    component="th"
-                                                    scope="row"
-                                                >
-                                                    <Typography noWrap>
-                                                        {getPercent(
-                                                            value.completedLength,
-                                                            value.length
-                                                        ).toFixed(2)}
-                                                        %
-                                                    </Typography>
-                                                </TableCell>
-                                            </TableRow>
-                                        );
-                                    })}
-                                </TableBody>
-                            </Table>
-                        </div>
+                        <TableVirtuoso
```

Comment:
需要覆盖一下 Table 元素，不然元素长度小的时候撑不起来。
![image](https://user-images.githubusercontent.com/16058869/175532101-1c5ccf52-61e5-4979-a255-0a9026ca2180.png)

```suggestion
                        <TableVirtuoso
                        components={{
                                Table: (props) => (
                                    <Table
                                        {...props}
                                    />
                                ),
                            }}
```